sidebar.jsx

```diff
@@ -1,113 +1,285 @@
 import React from 'react';
+import axios from 'axios';
 
-var Sidebar = ({product, seller}) => {
+/* const Sidebar = ({ product, seller }) => {

```

Comment:
I was a little confused and thought that you declared Sidebar twice, until I noticed the comment notation. If you're sure you don't need this anymore, maybe you could get rid of this comment?

Reply:
Got it. I just removed that part, so it won't show up next time in the pull request.